srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -2815,14 +2864,14 @@ CONFIG_BE2NET_LANCER=y
 CONFIG_BE2NET_SKYHAWK=y
 CONFIG_NET_VENDOR_EZCHIP=y
 CONFIG_NET_VENDOR_GOOGLE=y
-CONFIG_GVE=m
+# CONFIG_GVE is not set
 CONFIG_NET_VENDOR_HUAWEI=y
 CONFIG_HINIC=m
 CONFIG_NET_VENDOR_I825XX=y
 CONFIG_NET_VENDOR_INTEL=y
-CONFIG_E100=y
-CONFIG_E1000=y
-CONFIG_E1000E=y
+CONFIG_E100=m
+CONFIG_E1000=m
+CONFIG_E1000E=m
```

Comment:
and this is wrong, I made this one explicitely built-in to make qemu work ootb a few years ago...

Reply:
If you want more details, there's a qemu platform config that contains this eth dev... found out years ago that making it builtin we didn't need an initramfs...

Reply:
This got changed during Linux 5.1 upgrade, but sounds reasonable.